codes/classical/q-ary_digits/justesen.yml

```diff
@@ -0,0 +1,118 @@
+  # encoders:
+  #   - 'Specific description of a process that makes the states, usually for quantum codes.'
+  #   - 'Unitary circuit of depth ... \cite{arxiv:old-paper}.'
+  #   - 'Measurement-based preparation ... with ancilla overhead of ... .'
+  #   - 'Leave discussion of fault tolerance to fault-tolerance field.'
+
+  # # Not all fields are indexed by a dash
+  # transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  # general_gates:
+  #   - 'Universal gate set achieved by either additional ... gate.'
+  #   - 'Magic-state distillation protocols'
+  #   - 'kth Clifford hierarchy gates obtained by ... circuits'
+
+  decoders:
+      - 'generalized minimum distance decoding \cite{doi: 10.1109/TIT.1972.1054893}'
+  #   - 'MWPM decoding algorithm \cite{doi:good-paper} with ... overhead.'
+  #   - 'Just-in-time decoder with ... \cite{arxiv:awesome-paper}.'
+
+  # fault_tolerance:
+  #   - 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+  #   - 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+  #   - 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+  #   - 'Pieceable fault tolerance.'
+
+  # code_capacity_threshold:
+  #   - '\(1.5%\) error-correction threshold against some noise with *noiseless* decoder of some complexity \cite{arxiv:paper}.'
+  #   - '\(5.2\%\) asymptotic upper bound from some estimation algortihm (Monte Carlo, e.g.) that may or may not have a feasible decoder.'
+
+  # threshold:
+  #   - '\(0.3\%\) error-correction threshold ... with *noisy* ... decoder of some complexity \cite{doi:good-paper}.'
+  #   - '\(10^{-5}\) computational threshold using concatenated scheme under ... noise with overhead of ... '
+  #   - '\(4.5\%\) asymptotic lower bound on fault-tolerant threshold with ... noise, ... overhead, and ... complexity.'
+  #   - '\(6.7\%\) threshold using magic-state distillation protocols assuming ... Clifford gates.'
+
+  # # Propose a field that you feel is applicable to a large and interesting class of codes
+  # #some_other_field:
+  # # - 'detail'
+
+realizations: ''
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  # - 'Code used in DVDs \cite{doi:####...}, 5G, etc.'
+  # - 'Realized in trapped-ion quantum devices \cite{arXiv:####.#####}, etc.'
+
+notes: 'The code can be improved and extend the range of R from 0 to 1 by puncturiing the code (by erasing s digits from each inner code), resulting in a code of length n=(2m-s)N, rate R=mk/(2m-s)N. The lower bound of the distance of the punctured code
+        approaches d=(1-R/r)H^{-1}(1-R) as m goes to infinity, where r is the solution to R=r^2/(1+log(1-H^{-1}(1-r))), and H being the binary entropy function.'
```

Comment:
Line 106 describes an intermediate result. Please replace the code parameters with the parameters cited in MacWilliams and Sloane, Thm.10.12, page 214.